llm/server.go

```diff
@@ -1439,6 +1439,7 @@ func (d DoneReason) String() string {
 // TokenLogprob represents log probability information for a single token alternative.
 type TokenLogprob struct {
 	Token   string  `json:"token"`
+	Bytes   []int   `json:"bytes,omitempty"`
```

Comment:
Possible to add some tests. A few I can think of here:

1. Generate/chat tests
2. Edit the integration test to check `bytes`

Reply:
Yes sounds good, will add